daphne/src/vdaf/prio3_test.rs

```diff
@@ -46,6 +48,16 @@ fn prepare_histogram() {
     .unwrap();
 }
 
+#[test]
+fn prepare_sum_vec() {
+    test_prepare(
+        &Prio3Config::SumVec { bits: 23, len: 3 },
+        DapMeasurement::U64Vec(vec![1, 0, 42]),
```

Comment:
I'd add `2^23 - 1` as an additional use case here.

Reply:
I added that as an extra test, and also added U128 as a separate test